packages/@okta/i18n/src/properties/login.properties

```diff
@@ -591,7 +591,10 @@ registration.error.userName.notUniqueWithinOrg=An account with that email alread
 
 # PIV Auth
 piv.card=PIV Card
+piv.cac.card=Sign in with PIV / CAC card
```

Comment:
nit: we can group the `piv.cac.*` strings together right after this block of `piv.card.*`, just to have the related strings together, instead of all mixed up